modules/administration/pages/mgr-sync.adoc

```diff
@@ -0,0 +1,49 @@
+[[mgr.sync]]
+= Using ``mgr-sync``
+
+The ``mgr-sync`` tool is used at the command prompt.
+It provides functions for using {productname} that are not always available in the {webui}.
+The primary use of ``mgr-sync`` is to connext to the {scc} and retrieve product and package information for installed products.
+
+This tool is designed for use with a {suse} support subscription.
+It is not required for open source distributions, including {opensuse}, {centos}, and {ubuntu}.
+
+The available commands and arguments for ``mgr-sync`` are listed in this table.
+Use this syntax for ``mgr-sync`` commands:
+
+----
+mgr-sync [-h] [--version] [-v] [-s] [-d {1,2,3}] {list,add,refresh,delete}
+----
+
+[[mgr-sync-commands]]
+[cols="1,1,1", options="header"]
+.mgr-sync Commands
+|===
+| Command | Description | Example Use
+| list | List channels, organization credentials, or products | ``mgr-sync list``
+| add | Add channels, organization credentials, or products | ``mgr-sync add <channel_name>``
+| refresh | Refresh the local copy of products, channels, and subscriptions | ``mgr-sync refresh``
+| delete | Delete existing SCC organization credentials from the local system | ``mgr-sync delete``
+| sync | Synchronize all channels if left blank, or specify a specific channel | ``mgr-sync sync <channel_name>``
+|===
+
+
+
+[[mgr-sync-optional-args]]
+[cols="1,1,1,1", options="header"]
+.mgr-sync Optional Arguments
+|===
+| Option | Abbreviated option | Description | Example Use
+| help | ``h`` | Display the command usage and options | ``mgr-sync --help``
+| version | N/A | Display the currently installed version of ``mgr-sync`` | ``mgr-sync --version``
+| verbose | ``v`` | Provide verbose output | ``mgr-sync --verbose refresh``
+| store-credentials | ``s`` | Store credentials a local hidden file | ``mgr-sync --store-credentials``
+| debug | ``d`` | Log additional debugging information. Requires a level of 1, 2, 3. 3 provides the highest amnount of debugging information | ``mgr-sync -d 3 refresh``
+| no-sync | N/A | Use with the ``add`` command to add products or channels without beginning a synchronization | ``mgr-sync --no-sync add <channel_name>``
```

Comment:
The subcommands have a lot of more options. Not sure how to describe them here.

Reply:
That depends how important it is that we include them, I guess.

Reply:
What about adding a note with some thing like. Options specific for commands can be found by calling 

`mgr-sync <command> --help`